docs/src/.vitepress/config.mts

```diff
@@ -12,7 +27,11 @@ export default defineConfig({
   lastUpdated: true,
   cleanUrls: true,
   outDir: 'REPLACE_ME_DOCUMENTER_VITEPRESS', // This is required for MarkdownVitepress to work correctly...
-  head: [['link', { rel: 'icon', href: '/DocumenterVitepress.jl/dev/favicon.ico' }]],
+  head: [
+    ['link', { rel: 'icon', href: '/DocumenterVitepress.jl/dev/favicon.ico' }],
+    ['script', {src: '/versions.js'}],
```

Comment:
@asinghvi17  not here? I think I just need to add base here, isn't?

Reply:
Yes that should also work!

Reply:
yeah, that doesn't work. It populates the menu, but the links are wrong.

Reply:
Mmm, not sure how to propagate base to the Vue component.  If I'm not too tired, I will check again later. It will be good to have this finally in the next release,  which will be breaking either way. So, a big bump will also be needed. Someone that knows more about it, please commit the appropriate bump.